lessons/bn/basics/basics.md

```diff
@@ -1,19 +1,19 @@
 %{
-  version: "1.0.0",
+  version: "1.3.0",
   title: "ব্যাসিকস",
   excerpt: """
-  ইন্সটলেশন, ইন্টারএ্যাক্টিভ প্রম্পট, ব্যাসিক ডাটা টাইপ ও ব্যাসিক অপারেশন
+  শুরুর কথা, ব্যাসিক ডাটা টাইপ ও ব্যাসিক অপারেশন
   """
 }
 ---
 
-## ইন্সটলেশন ও ইন্টারএ্যাক্টিভ মোড
+## শুরুর কথা
+## এলিক্সির ইন্সটলেশন
 
-### ইন্সটলেশন
 
-শুরুতেই আমাদের এলিক্সির ইন্সটল করতে হবে । আলাদা আলাদা অপারেটিং সিস্টেমের জন্য ইন্সটলেশনের বিস্তারিত নির্দেশনা পাওয়া যাবে এলিক্সির এর অফিশিয়াল ওয়েব সাইটের এই সেকশনে - [Installing Elixir](http://elixir-lang.org/install.html) । এই গাইডলাইন অনুসরন করে প্রথমে এলিক্সির ইন্সটল করে নিন । 
+শুরুতেই আমাদের এলিক্সির ইন্সটল করতে হবে । আলাদা আলাদা অপারেটিং সিস্টেমের জন্য ইন্সটলেশনের বিস্তারিত নির্দেশনা পাওয়া যাবে এলিক্সির এর অফিশিয়াল ওয়েব সাইটের এই সেকশনে - [Installing Elixir](http://elixir-lang.org/install.html) । এই গাইডলাইন অনুসরন করে প্রথমে এলিক্সির ইন্সটল করে নিন ।
```

Comment:
Welcome to elixirschool, @m4hi2 bhai! Just nitpicking can you also remove the extra space in between the word and the Bangla full stop symbol? 
For example: `হবে ।` should be `হবে।` 
There is also a small spelling mistake: `অনুসরন` should be `অনুসরণ` as far as I know. 
Thanks for your valuable contribution! 😄

Reply:
@wasi0013 bhai, thanks for the suggestions. these are dinosaurs from past localization efforts. I'll fix these and be mindful of issues like this in the future. Thanks again.